pkg/volume/csi/csi_plugin_test.go

```diff
@@ -1,279 +0,0 @@
-/*
-Copyright 2017 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package csi
-
-import (
-	"fmt"
-	"os"
-	"path"
-	"testing"
-
-	api "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/resource"
-	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
-	fakeclient "k8s.io/client-go/kubernetes/fake"
-	utiltesting "k8s.io/client-go/util/testing"
-	"k8s.io/kubernetes/pkg/volume"
-	volumetest "k8s.io/kubernetes/pkg/volume/testing"
-)
-
-// create a plugin mgr to load plugins and setup a fake client
-func newTestPlugin(t *testing.T) (*csiPlugin, string) {
```

Comment:
Why are plugin tests deleted ?

Reply:
all CSI tests are using setupClient func from csi_client_tests which is using fake package to mock the driver. So far I could not figure out a way to make it  working with new Identity/Node/Controller clients. 
I always get:
```
panic: interface conversion: csi.NodeClient is *csi.nodeClient, not *fake.NodeClient [recovered]
        panic: interface conversion: csi.NodeClient is *csi.nodeClient, not *fake.NodeClient
```

Reply:
Looks like the clients are now being passed around by reference, so the test code should be updated to match that.